core/src/trezor/ui/layouts/bolt/__init__.py

```
@@ -674,46 +678,46 @@ async def _confirm_ask_pagination(
-            confirm_more_layout, br_name, br_code, raise_on_cancel=None
-        )
-        if result is CONFIRMED:
-            return
+            result = await interact(
+                confirm_more_layout, br_name, br_code, raise_on_cancel=None
+            )
+            if result is CONFIRMED:
+                return
 
-    assert False
+        assert False
```

Ruff (0.15.17)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)